### PR TITLE
Maryia/CanvasMenu (sidebar for responsive mode) + Footer + Tests

### PR DESCRIPTION
**Pull request by maryia-deriv-archive**

### CODE REVIEW

**Comment by maryia-deriv-archive** [comment]
still working on Footer & tests

**Comment by maryia-deriv-archive** [comment]
still working on tests

File: src/__tests__/HomePageTests/GetStartedCard.test.tsx [hunk 1]
```diff
@@ -4,19 +4,43 @@ import GetStartedCard from "../../components/Home/Sections/GetStarted/GetStarted
...
+        render(
+            <GetStartedCard
+                number="1"
+                href="https://deriv.com/"
+                icon="test.svg"
+                title="test title"
+                content="test content"
+            />
+        );
         expect(screen.getByAltText("test icon")).toBeInTheDocument();
```
Comment (george-usynin-deriv) [inline src/__tests__/HomePageTests/GetStartedCard.test.tsx:28]: don't repeat yourself, you can create a variable props, and then spread it to each component:
const {
number="1"
href="https://deriv.com/"
icon="test.svg"
title="test title"
content="test content"
} = props

 <GetStartedCard {...props} />

Comment (george-usynin-deriv) [reply src/__tests__/HomePageTests/GetStartedCard.test.tsx:28]: This message is for Henadziy! Not for Maria !! 😊

**Review by george-usynin-deriv — APPROVED** [review]

**Pull request merged**